filebeat/module/apache2/access/ingest/default.json

```diff
@@ -31,19 +31,13 @@
   }, {
     "user_agent": {
       "field": "apache2.access.agent",
-      "target_field": "apache2.access.user_agent",
-      "ignore_failure": true
-    }
-  }, {
-    "rename": {
-      "field": "apache2.access.agent",
-      "target_field": "apache2.access.user_agent.original",
+      "target_field": "user_agent",
       "ignore_failure": true
     }
   }, {
     "geoip": {
-      "field": "apache2.access.remote_ip",
-      "target_field": "apache2.access.geoip"
+      "field": "source.ip",
+      "target_field": "source.geoip"
```

Comment:
The field is `source.geo`, without the "ip". Geo information can come from other means than geoip ;-)

Reply:
yep, old habit :-(